dot/network/sync_test.go

```diff
@@ -335,6 +262,8 @@ func TestSyncQueue_HandleBlockAnnounce(t *testing.T) {
 }
 
 func TestSyncQueue_ProcessBlockRequests(t *testing.T) {
+	//t.Skip() // TODO: fix this
+
```

Comment:
Does this still need to be fixed?

Reply:
oh that is already fixed I forgot to remove that, will remove